packages/@n8n/task-runner/src/config/sentry-config.ts

```diff
@@ -0,0 +1,21 @@
+import { Config, Env } from '@n8n/config';
+
+@Config
+export class SentryConfig {
+	/** Sentry DSN */
```

Comment:
```suggestion
	/** Sentry data source name */
```

Reply:
Oh I had no idea what that was an acronym for. Even Sentry only talks about DSNs, so it would be confusing to not use that

<img width="170" alt="image" src="https://github.com/user-attachments/assets/d7b280e9-bc8a-47d7-bfcc-c583cf32e4a8">

<img width="297" alt="image" src="https://github.com/user-attachments/assets/928c9a0a-186f-4e18-baa6-875b681da307">

Reply:
We could always do `DSN (data source name)`